### ISSUE TITLE
Tron audio issues

### ISSUE THREAD

**Issue by shertz43** [MiSTer-devel/Arcade-MCR2_MiSTer#3]
There is suppose to be no audio in attract mode.

Tron lightcycle crash is not correct

Tank background 'rumble' sound is not right.

**Comment by DNCrosby** [comment]
Yeah it seems like R-135 isn't modeled correctly which pulls the volume of attract mode to not be heard, and if removed attract mode will play sound. A lot of people add a switch or solder to one of the un-used dips to toggle this...
